cmd/icinga-notifications-daemon/main.go

```diff
@@ -15,10 +16,17 @@ import (
 
 func main() {
 	var configPath string
+	var showVersion bool
 
 	flag.StringVar(&configPath, "config", "", "path to config file")
+	flag.BoolVar(&showVersion, "version", false, "print version")
 	flag.Parse()
 
+	if showVersion {
+		internal.Version.Print()
```

Comment:
Good that I built `github.com/icinga/icingadb/pkg/version` somewhat reusable. Not so good that I forget that you might want to replace the project name if you actually reuse it. :facepalm: 

So probably copy [these few lines](https://github.com/Icinga/icingadb/blob/c20f066bd87528960ec095c2e6fd88d014156b6c/pkg/version/version.go#L83-L90) here for now (I don't really care about the system information at the moment).